README.md

```diff
@@ -244,12 +244,18 @@ The Moodle setup is very similar to both the examples outlined above.
    config.yaml:
 
 ```yaml
-    auth:
-      type: "lti"
-      lti:
-          consumers: { 
-              "client-key"": "client-secret"
-              }
+  hub:
+    config:
+      LTIAuthenticator:
+        consumers: { 
+          "client-key"": "client-secret"
+        }
+      JupyterHub:
+        authenticator_class: ltiauthenticator.LTIAuthenticator
+    extraConfig:
+      myAuthConfig: |
+        from ltiauthenticator import LTIAuthenticator
+        c.JupyterHub.authenticator_class = LTIAuthenticator
```

Comment:
```suggestion
  hub:
    config:
      LTIAuthenticator:
        consumers:
          client-key: client-secret
      JupyterHub:
        authenticator_class: ltiauthenticator.LTIAuthenticator
```

Reply:
@consideRatio I don't think this would work with the JupyterHub helm chart. From what I understand we would need to add an entrypoint to the `jupyterhub-ltiauthenticator` package `setup.py` so that the `'ltiauthenticator.LTIAuthenticator'` is recognized with the helm's config as a valid authenticator class.

But maybe I'm missing something 😄

Reply:
I believe these are equivalent configs, juputerhub will attempt to load the module and instanciate the class based on the string it is passed by itself.

Reply:
If im wrong, you should not have a duplicate entry for configuring the authenticator class in hub.config

Reply:
hub.config is just a way to pass values to c.[some class here].[some config name here] just like you so via extraConfig.

Reply:
Via hub.config you can only pass a string, but i think that should be okay. No matter what, we should only attempt to set it once.

Reply:
I think you only need the entrypoint if you want to say 'lti' instead of  ltiauthenticator.LTIAuthenticator

Reply:
Thanks for the clarification @consideRatio and @yuvipanda!